wrappers/ortools.rb

```diff
@@ -75,6 +75,7 @@ def solver_constraints
         :assert_route_date_or_indice_if_periodic,
         :assert_not_too_many_visits_in_route,
         :assert_no_route_if_schedule_without_periodic_heuristic,
+        # :assert_no_overall_duration, # Requires a complete rework
```

Comment:
So that if "one day" we attack the todo's it'll be visible to make us unhappy :)  
```suggestion
        # :assert_no_overall_duration, # TODO: Requires a complete rework
```

Reply:
I have the impression that you pushed before finishing the edits :D